app/Model/CoEnrollmentFlow.php

```diff
@@ -847,85 +847,87 @@ public function configuredSteps($id) {
   /**
    * Duplicate an existing Enrollment Flow.
    *
-   * @since  COmanage Registry v0.9.2
```

Comment:
The implementation may be changing but the function was still introduced in v0.9.2.

Reply:
Done